Tests/FluentAssertions.Specs/Xml/XAttributeAssertionSpecs.cs

```diff
@@ -40,6 +40,35 @@ public void When_asserting_an_xml_attribute_is_equal_to_a_different_xml_attribut
                 $"Expected theAttribute to be {otherAttribute} because we want to test the failure message, but found {theAttribute}.");
         }
 
+        [Fact]
+        public void When_xml_attribute_is_null_then_be_should_fail()
+        {
+            XAttribute theAttribute = null;
+
+            // Act
+            Action act = () =>
+                theAttribute.Should().Be(new XAttribute("name", "value"), "we want to test the failure {0}", "message");
+
+            // Assert
+            act.Should().Throw<XunitException>()
+                .WithMessage(
+                    "Expected the attribute to be name=\"value\" *failure message*, but theAttribute is <null>.");
+        }
+
+        [Fact]
+        public void When_asserting_àttribute_should_be_equal_to_null_it_should_throw()
```

Comment:
You added (accidentally?) an `à` instead of `a`.

Reply:
Yeah, must have hit it while coding in bed :)